Content/LeagueSandbox-Scripts/Buffs/Global/Recall.cs

```diff
@@ -55,16 +58,20 @@ public void OnUpdateMoveOrder(IObjAiBase unit)
             var buff = unit.GetBuffWithName("Recall");
             if (buff != null)
             {
-                buff.DeactivateBuff();
+                if (unit.MoveOrder != OrderType.Hold && unit.MoveOrder != OrderType.Stop)
+                {
+                    buff.DeactivateBuff();
+                }
+                else
+                {
+                    // After the callback ends, it will remove the listener, so we make a new one before the callback ends.
+                    ApiEventManager.OnUnitUpdateMoveOrder.AddListener(this, unit, OnUpdateMoveOrder, true);
```

Comment:
You can avoid it by simply not changing the true to false for 'singleinstance' callback.